README.md

```diff
@@ -2,3 +2,13 @@
 
 A set of packages to help build Git-backed applications.
 Weave GitOps Toolkit builds on top of the [Kubernetes API Machinery](https://github.com/kubernetes/apimachinery).
+
+## Getting Help
+
+If you have any questions about, feedback for or problems with `ignite`:
```

Comment:
Are we planning to provide support for this tool via the ignite slack channel?  I think that's OK but we should consider changing this message to say something about that.  i.e. "for help with gitops-toolkit, post a message on the `ignite` channel in the Weave Users Slack..."